curation_concerns-models/app/models/concerns/curation_concerns/file_set/derivatives.rb

```diff
@@ -16,11 +17,15 @@ def create_derivatives(filename)
         when *self.class.pdf_mime_types
           Hydra::Derivatives::PdfDerivatives.create(filename,
                                                     outputs: [{ label: :thumbnail, format: 'jpg', size: '338x493', url: derivative_url('thumbnail') }])
+          Hydra::Derivatives::FullTextExtract.create(filename,
+                                                     outputs: [{ url: uri, container: "extracted_text" }])
         when *self.class.office_document_mime_types
           Hydra::Derivatives::DocumentDerivatives.create(filename,
                                                          outputs: [{ label: :thumbnail, format: 'jpg',
                                                                      size: '200x150>',
                                                                      url: derivative_url('thumbnail') }])
+          Hydra::Derivatives::FullTextExtract.create(filename,
+                                                     outputs: [{ url: uri, container: "extracted_text" }])
         when *self.class.audio_mime_types
           Hydra::Derivatives::AudioDerivatives.create(filename,
                                                       outputs: [{ label: 'mp3', format: 'mp3', url: derivative_url('mp3') },
```

Comment:
Pushing large files over the network (videos and audios in particular) is something I'd like to avoid.  I've also seen missing class issues when trying to do a TIFF ( I think @cam156 confirmed scholarsphere has this problem)